src/guides/v2.3/performance-best-practices/deployment-flow.md

```diff
@@ -38,22 +28,70 @@ When you preprocess and compile dependency injection (DI) instructions, Magento:
 *  Creates autogenerated files (including proxies, factories, etc.)
 *  Stores compiled data and configuration in a cache that saves up to 25% of time on requests processing
 
-Run the following command to preprocess and compile DI:
+To preprocess and compile DI instructions:
 
 ```bash
 bin/magento setup:di:compile
 ```
 
-After compilation completes, confirm that [APCu is enabled]({{ page.baseurl }}/performance-best-practices/software.html#php-settings), then run the following command:
+## Update the autoloader
+
+After compilation completes, confirm that [APCu is enabled]({{ page.baseurl }}/performance-best-practices/software.html#php-settings) and update the autoloader:
+
+To update the autoloader:
+
+{:.bs-callout-info}
+The `-o` option converts PSR-0/4 autoloading to classmap to get a faster autoloader. The `--apcu` option uses APCu to cache found/not-found classes.
 
 ```bash
 composer dump-autoload -o --apcu
 ```
 
+If you plan on updating the autoloader, you must run the following commands in order:
+
+```bash
+composer install
+```
+
+```bash
+bin/magento setup:di:compile
+```
+
+```bash
+composer dump-autoload -o
+```
+
+```bash
+bin/magento setup:static-content:deploy
+```
+
+## Deploy static content
+
+Deploying static content causes Magento 2 to perform the following actions:
+
+*  Analyze all static resources
+*  Perform merge, minimization, and bundling of content
+*  Read and process theme data
+*  Analyze theme fallback
+*  Store all processed and materialized content to specific folder for further usage
+
+If your static content is not deployed, Magento performs all listed operation on the fly, leading to a significant increase in response time.
+
+You can use a variety of options to customize deployment operations based on store size and fulfillment needs. The most common is the compact deploy strategy. See [Static files deployment strategies]({{ page.baseurl }}/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html)
+
+To deploy static content:
+
+```bash
+bin/magento setup:static-content:deploy
+```
+
 This command allows Composer to rebuild the mapping to project files so that they load faster.
 
 ## Set production mode
 
+{:.bs-callout-info}
+Setting production mode automatically runs `setup:di:compile` and `setup:static-content:deploy`.
```

Comment:
I have a little problem with this sentence. Is something setting the production mode automatically? Or is the production mode automatically running the commands? It can read both. Maybe you need an article in front of production mode? Or not? Depends on how you want it, but something is missing.

"Setting the production mode automatically..."
"Production mode automatically..."

Reply:
Revised